content/en/install.md

```diff
@@ -3,101 +3,149 @@ title: Installation
 
-Other options include:
+    python -m pip install scipy
 
-- [WinPython](https://winpython.github.io): Another free distribution
-  including scientific packages and the Spyder IDE; Windows only, but
-  more actively maintained and supports the latest Python 3 versions.
-- [Pyzo](https://pyzo.org): A free distribution based on Anaconda
-  and the IEP interactive development environment; Supports Linux,
-  Windows, and Mac.
+[venv-guide]: https://packaging.python.org/en/latest/tutorials/installing-packages/#creating-virtual-environments
+[SciPy from PyPI]: https://pypi.org/project/scipy/
+[`pip`]: https://pip.pypa.io/en/stable/getting-started/
 
-<a name="pip-install"></a>
+Alternatively, you can install [SciPy from conda-forge] using a [Conda]-based
+environment manager. You can install Python itself, as well as packages in
+languages other than Python, as Conda packages.
 
-# Installing with Pip
+[Miniforge] is the recommended way to install `conda` and [`mamba`],
+two Conda-based environment managers. You can then install SciPy as follows:
 
-You can install SciPy from PyPI with `pip`:
+    conda install scipy # or
+    mamba install scipy
 
-    python -m pip install scipy
+[SciPy from conda-forge]: https://anaconda.org/conda-forge/scipy
+[Miniforge]: https://conda-forge.org/download/
+[Conda]: https://docs.conda.io/projects/conda/en/latest/index.html
+[`mamba`]: https://mamba.readthedocs.io/en/latest/
+
+<a name="project-based"></a>
+
+## Project-based workflows
+
+In project-based workflows, a project is a directory containing a manifest
+file describing the project, a lock-file describing the exact dependencies
+of the project, and the project's (potentially multiple) environments.
+These workflows provide reproducibility benefits over environment-based workflows.
+
+You can add [SciPy from PyPI] to a [`uv`] project:
+
+    uv add scipy
+
+You can also install Python itself with `uv`.
+
+[`uv`]: https://docs.astral.sh/uv/
+
+If you work with non-Python packages, you may prefer to install SciPy as
+a Conda package, so that you can use the same workflow for packages which
+are not available on PyPI.
 
-<a name="conda-install"></a>
+You can add [SciPy from conda-forge] to a [`pixi`] project:
 
-# Installing via Conda
+    pixi add scipy
 
-You can install SciPy from the `defaults` or `conda-forge` channels with `conda`:
+Pixi can also install packages from PyPI, using `uv` under the hood.
 
-    conda install scipy
+[`pixi`]: https://pixi.sh/latest/
 
-<a name="package_manager"></a>
+<a name="system-package-managers"></a>
 
-# Install system-wide via a package manager
+## Installing system-wide via a system package manager
 
 System package managers can install the most common Python packages.
 They install packages for the entire computer, often use older versions,
-and don't have as many available versions.
+and don't have as many available versions. They are not the recommended
+installation method, but they are the simplest.
```

Comment:
```suggestion
installation method.
```

Causes a lot of headaches, so wouldn't give any suggestion that this route is simple. E.g., on Debian you run into issues because you can't subsequently pip install without special flags.

Reply:
It's pretty short, and it covers a couple of common use-cases, so I think OK to keep for now.